projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/components/google-drive-disconnect-button.tsx

```diff
@@ -0,0 +1,40 @@
+import apiFetch from '@wordpress/api-fetch';
+import { Button } from '@wordpress/components';
+import { useEffect, useState } from '@wordpress/element';
+import { __, _x } from '@wordpress/i18n';
+
+type Props = {
+	onDisconnected: () => void;
+	isConnected: boolean;
+};
+
+export default function GoogleDriveDisconnectButton( { onDisconnected, isConnected }: Props ) {
+	const [ isToggling, setIsToggling ] = useState( false );
+
+	useEffect( () => {
+		setIsToggling( false );
+	}, [ isConnected ] );
+
+	const handleClick = () => {
+		setIsToggling( true );
+		apiFetch( { method: 'DELETE', path: '/wp/v2/feedback/integrations/google-drive' } )
+			.then( ( response: { deleted: boolean } ) => {
+				if ( response.deleted ) {
+					onDisconnected();
+				} else {
+					setIsToggling( false );
+				}
+			} )
+			.catch( () => {
+				setIsToggling( false );
+			} );
+	};
+
+	return (
+		<Button variant="link" onClick={ handleClick } disabled={ isToggling }>
+			{ isToggling
+				? __( 'Disconnecting…', 'jetpack-forms' )
+				: _x( 'Disconnect Google Drive', '', 'jetpack-forms' ) }
```

Comment:
Same as https://github.com/Automattic/jetpack/pull/45648/files#r2468384151


```suggestion
				: __( 'Disconnect Google Drive', 'jetpack-forms' ) }
```

Reply:
Done.